docs/api/drivers/driver_interfaces.md

```diff
@@ -72,7 +72,7 @@ class LEDAnimator {
     }
 
     LEDAnimator(mbed::interface::DigitalOut *red, mbed::interface::DigitalOut *green, mbed::interface::DigitalOut *blue) : _red(red), _green(green), _blue(blue), _internally_created(false) {
-    ...
```

Comment:
was the ... intended to show that you can add your own code at this point? perhaps rather than simply leave blank, we should add some appropriate code comment on this line?

Reply:
yes, I guess so the `…` mean your own code. 
However this is not standard c/cpp syntax. the snippets won't compile. one task the GHA is doing is trying to complier those snippets make sure they are valid. so we have to change this? 
that about change that to a comment line?
```
// your own code
```

Reply:
I think this is fine, though I am not 100% sure of the intention of this part of the example code. Worth checking with @pan- who reviewed this new chapter originally?

Reply:
Based on the example code in the next chapter, perhaps this is sufficient / suitable for the comment?

// do something